web/src/views/events/EventView.tsx

```diff
@@ -611,23 +611,34 @@ function DetectionReview({
 
   // keyboard
 
-  useKeyboardListener(["a", "r"], (key, modifiers) => {
+  useKeyboardListener(["a", "r", "PageDown", "PageUp"], (key, modifiers) => {
     if (modifiers.repeat || !modifiers.down) {
       return;
     }
 
-    if (key == "a" && modifiers.ctrl) {
-      onSelectAllReviews();
-    }
-
-    if (key == "r" && selectedReviews.length > 0) {
-      currentItems?.forEach((item) => {
-        if (selectedReviews.includes(item.id)) {
-          item.has_been_reviewed = true;
-          markItemAsReviewed(item);
+    switch (key) {
+      case "a":
+        if (modifiers.ctrl) {
+          onSelectAllReviews();
         }
-      });
-      setSelectedReviews([]);
+        break;
+      case "r":
+        if (selectedReviews.length > 0) {
+          currentItems?.forEach((item) => {
+            if (selectedReviews.includes(item.id)) {
+              item.has_been_reviewed = true;
+              markItemAsReviewed(item);
+            }
+          });
+          setSelectedReviews([]);
+        }
+        break;
+      case "PageDown":
+        contentRef.current?.scrollBy({ top: 600, behavior: "smooth" });
```

Comment:
Rather than using a static value, we probably should take into account the user's screen height. Something like `contentRef.current.clientHeight` plus or minus some pixels would probably be better.